perf/src/main/scala/cromwell/perf/CompareMetadata.scala

```diff
@@ -18,13 +21,34 @@ import io.circe.java8.time.decodeOffsetDateTimeDefault
 object CompareMetadata extends App with StrictLogging{
   private  val REGRESSION_CONST = 1.1
 
-  def parseMetadataFromFile(filePath: String): Either[circe.Error, Metadata] = {
+
+  def parseMetadataFromLocalFile(filePath: String): Either[circe.Error, Metadata] = {
     val metadataFile = File(filePath)
     val metadataFileContent = metadataFile.contentAsString
 
     decode[Metadata](metadataFileContent)
   }
 
+  def parseMetadataFromGcsFile(gcsUrl: String): Either[circe.Error, Metadata] = {
+    val gcsUrlArray = gcsUrl.replace("gs://", "").split("/", 2)
+    val fileToBeLocalized = gcsUrlArray(1)
+    val gcsBucket = gcsUrlArray(0)
+
+    val credentials = GoogleCredentials.fromStream(new FileInputStream("/resources/cromwell-perf-service-account.json"))
```

Comment:
I think this `credentials` should come in as a function arg, and the file path string should come in via a Command line argument to  `CompareMetadata`